GCM/App/init/ScenesInitializerImpl.java

```diff
@@ -10,6 +10,12 @@
 import javafx.scene.Scene;
 import mainApp.GcmClient;
 import mainApp.SceneNames;
+<<<<<<< HEAD
```

Comment:
This is what a merge conflict leaves and it will fail to compile.

- Solve your merge conflict all the way through using beyond compare
- Check that the code compiles before pushing anything